observability/network_observability/installing-operators.adoc

```diff
@@ -24,9 +24,9 @@ include::modules/network-observability-loki-install.adoc[leveloffset=+1]
 include::modules/network-observability-loki-secret.adoc[leveloffset=+2]
 [role="_additional-resources"]
 .Additional resources
-* xref:../../observability/network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[Flow Collector API Reference] 
+* xref:../../observability/network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[Flow Collector API Reference]
 * xref:../../observability/network_observability/configuring-operator.adoc#network-observability-flowcollector-view_network_observability[Flow Collector sample resource]
-* xref:../../observability/logging/log_storage/installing-log-storage.adoc#logging-loki-storage_installing-log-storage[Loki object storage]
+//* xref :../../observability/logging/log_storage/installing-log-storage.adoc#logging-loki-storage_installing-log-storage[Loki object storage]
```

Comment:
So for 4.17+, /observability/logging is commented out of the topic map, I'd set up the logging folders so that we could drop the old logging stack docs once they hit end of life without actually removing any files from the repo. So while any xrefs to modules with a `/observability/logging` path will not work as xrefs, the modules themselves are still there. You are welcome to claim any of them for NetObserv and edit them as you wish.